tensorboard/webapp/app_routing/internal_utils.ts

```diff
@@ -178,6 +178,7 @@ export function getDeepLinkGroup(routeKind: RouteKind): DeepLinkGroup | null {
     case RouteKind.COMPARE_EXPERIMENT:
       return DeepLinkGroup.DASHBOARD;
     case RouteKind.UNKNOWN:
+    case RouteKind.CARD:
```

Comment:
nit: a-z order

Reply:
We still want to preserve the deep link for card route. Can you move it to line 178? You will need to address this internally. I guess it's not too much work but if it turns out to be. We can then come back to do it here.